source/_patterns/01-elements/input/input.demonstration.scss

```diff
@@ -1,9 +1,21 @@
+@use "@db-ui/base/build/scss/tonality" as *;
+
 [id|="elements-input"] {
+	dl {
+		display: grid;
+		grid-template-columns: 1fr 5fr;
+		align-items: center;
+	}
+
 	dd {
-		margin-bottom: 2rem;
+		margin: 1rem 0;
 	}
 }
 
 .elm-input {
 	max-width: 30rem;
 }
+
+.pl-c-pattern {
+	@extend %db-ui-regular;
+}
```

Comment:
what about moving this to `source/css/pattern-scaffolding-project-specific.scss` instead as it's a quite global selector?

Reply:
good point, changed that.